### DISCUSSION TITLE
Silent Japanese streaming buffers return transcript "です。" or "の".

### DISCUSSION THREAD

**Discussion by FaizeKhan12** [deepgram/community#522] [General help] [answered]
### Which Deepgram product are you using?

Deepgram SDKs

#### Upon integration of the Deepgram SDK into our application, the streaming transcription functionality has proven to be robust and reliable. However, an anomaly arises when sending silent audio buffers while specifying Japanese as the language. Surprisingly, Deepgram transcribes these silent audio segments and returns unexpected results with the interim response, such as "です。" or "の".

**It is noteworthy that I have conducted tests with the English language, and the Deepgram SDK performs as expected in that context. The anomaly seems to be specific to the Japanese language transcription when processing silent audio buffers.**

### If possible, please attach your code or paste it into the text box.

***To provide clarity, here are the key properties set during the request creation:***

var options = new LiveTranscriptionOptions()
{
    Punctuate = true,
    Encoding = Deepgram.Common.AudioEncoding.MuLaw,
    SampleRate = 8000,
    SmartFormat = true,
    Language = "ja",
    Model = "enhanced",
    InterimResults = true,
};
_deepgramLive.StartConnectionAsync(options);

####Issue persist with all models available for Japanese.

#### Issue can be replicated with this audio. I was muting myself after saying the utterance which was getting transcribed correctly but after muting, Deepgram is sending this "です。" or "の".:
[Audio](https://storage.googleapis.com/deepgram_issue/Customer%20Audio%20-%20Japanese%20language%20detecting%20(IS).wav).

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jkroll-deepgram** [comment, marked as answer]
As we discussed, Deepgram is actively working on a couple of improvements to our Japanese models, with the aim of resolving this issue of silent audio falsely being transcribed with errant characters.
